CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 * [BUGFIX] Ingester: Flushing chunks via `/flush` endpoint could previously lead to panic, if chunks were already flushed before and then removed from memory during the flush caused by `/flush` handler. Immediate flush now doesn't cause chunks to be flushed again. Samples received during flush triggered via `/flush` handler are no longer discarded. #2778
```

Comment:
Can you add a `[CHANGE]` entry as well explaining the new label behaviour on metrics on the KV store.

Reply:
I added a `[CHANGE]` log that explain the `role` label to CHANGELOG.md